db/migrate/20211110152802_remove_image_column_from_monsters.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveImageColumnFromMonsters < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :monsters, :image, :string
```

Comment:
It looks like just images were removed, but we should remove votes as well

Reply:
Resolved